mime-types-redirector/mime-types.gemspec

```diff
@@ -0,0 +1,19 @@
+# -*- encoding: utf-8 -*-
+# stub: mime-types 3.1 ruby lib
+
+Gem::Specification.new do |s|
+  s.name = "mime-types"
+  s.version = "2.6.1"
+
+  s.required_rubygems_version = Gem::Requirement.new(">= 0") if s.respond_to? :required_rubygems_version=
+  s.require_paths = ["lib"]
+  s.authors = ["Nick LaMuro"]
+  s.date = "2017-03-27"
+  s.description = "Stub mime-types repo that redirects to mini-mime"
+  s.files = ["lib/mime-types.rb", "lib/mime/types.rb"]
+  s.required_ruby_version = Gem::Requirement.new(">= 2.0")
+  s.rubygems_version = "2.6.4"
+  s.summary = "Stub mime-types repo that redirects to mini-mime"
+
+  s.add_dependency 'mini_mime', '>= 0.1.1'
+end
```

Comment:
I'm wondering if we can just make this a released gem somehow, but I can't think of how you would make the Gemfile work. 🤔